### DISCUSSION TITLE
Mulit-line string in playground?

### DISCUSSION THREAD

**Discussion by Rec0iL99** [apollographql/apollo-server#6860] [Q&A] [answered]
Is it possible to pass a multi-line string as argument to a graphql mutation?

According to StackOverflow, people are recommending to wrap the multi-line string in `""" """`. I tried doing this, but it doesn't seem to help.

<img width="401" alt="Screenshot 2022-08-27 at 7 10 41 PM" src="https://user-images.githubusercontent.com/56463684/187032811-7f420d0d-94fb-47f9-901b-db6fd124c9e3.png">

**Comment by glasser** [comment, marked as answer]
GraphQL Playground is a project whose maintainers have officially retired it. We only provide an integration with Apollo Server to help folks upgrading from older versions of Apollo Server which only provided Playground as a UI not have to relearn a new UI immediately. If you're finding that this legacy project does not support basic GraphQL syntax (surprising but imaginable), I suggest switching to an actively maintained web interface like Apollo Sandbox. (I'm guessing you're still on Apollo Server 2 which has GraphQL Playground and several other outdated dependencies baked in; I strongly encourage upgrading!)

**Reply by Rec0iL99** [reply]
Got it! Need to start using Apollo Sandbox instead of Playground.

Thanks a lot for the reply!
